### PR TITLE
avm2: Implement `flash.ui.Keyboard`

### PR DESCRIPTION
**Pull request by Herschel**
Add `Keyboard` key code constants and stub out other properties.

### CODE REVIEW

File: core/src/events.rs [hunk 1]
```diff
@@ -336,11 +336,13 @@ pub enum KeyCode {
     Backspace = 8,
     Tab = 9,
     Return = 13,
+    Command = 15,
     Shift = 16,
     Control = 17,
     Alt = 18,
     Pause = 19,
     CapsLock = 20,
+    Numpad = 21,
```
Comment (torokati44) [inline core/src/events.rs:345]: Did you mean `NumLock` here?

Comment (torokati44) [reply core/src/events.rs:345]: Oh no, it's a modifier, or something like that, sorry: https://help.adobe.com/en_US/air/reference/html/flash/ui/Keyboard.html#NUMPAD
It was just weird at a first glance.

File: core/src/avm2/globals/flash/ui/keyboard.rs [hunk 2]
```diff
@@ -38,5 +40,352 @@ pub fn create_class<'gc>(mc: MutationContext<'gc, '_>) -> GcCell<'gc, Class<'gc>
...
+        ("STRING_DELETECHAR", ""),
+        ("STRING_PREV", ""),
+        ("STRING_NEXT", ""),
+        ("STRING_SELECT", ""),
+        ("STRING_EXECUTE", ""),
+        ("STRING_UNDO", ""),
+        ("STRING_REDO", ""),
+        ("STRING_FIND", ""),
+        ("STRING_HELP", ""),
+        ("STRING_MODESWITCH", ""),
```
Comment (torokati44) [inline core/src/avm2/globals/flash/ui/keyboard.rs:188]: Just a nit: These look... interesting... perhaps escaping them and writing them numerically would be nicer?

Comment (Herschel) [reply core/src/avm2/globals/flash/ui/keyboard.rs:188]: I agree; I copied them out of JPEXS and was too lazy to escape them :)

File: core/src/events.rs (hunk 1 above)
Comment (Herschel) [reply core/src/events.rs:345]: I did the same double take. It's possible we could give it a different name.

Comment (torokati44) [reply core/src/events.rs:345]: Or perhaps a comment like `// A meta-key used as modifier, not NumLock` would clear it up.

**Pull request merged**